server/src/main/java/com/objectcomputing/member/MemberProfile.java

```diff
@@ -0,0 +1,68 @@
+package com.objectcomputing.member;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.sql.Date;
+
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@Table(name ="member_profile")
+public class MemberProfile {
+
+    public MemberProfile() {}
+
+    public MemberProfile(String name,String role,Long pdlId,String location
+                        ,String workEmail, String insperityId, Date startDate,
+                        String bioText) {
+                        this.name=name;
+                        this.role=role;
+                        this.pdlId=pdlId;
+                        this.location=location;
+                        this.workEmail=workEmail;
+                        this.insperityId=insperityId;
+                        this.startDate=startDate;
+                        this.bioText=bioText;
+                        }
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
```

Comment:
Might I suggest using a UUID?
```
    @Id
    @Column(name="uuid")
    @AutoPopulated
    private UUID uuid;
```
Then you just define it as a `varchar` in the database.

Reply:
By the way, `@AutoPopulated` is a Micronaut annotation: `import io.micronaut.data.annotation.AutoPopulated;`

Reply:
Sure, I will change it to UUID.